platform/src/main/java/org/stellar/anchor/platform/controller/sep/Sep6Controller.java

```diff
@@ -90,7 +92,8 @@ public StartDepositResponse depositExchange(
       @RequestParam(value = "account") String account,
       @RequestParam(value = "memo_type", required = false) String memoType,
       @RequestParam(value = "memo", required = false) String memo,
-      @RequestParam(value = "type") String type,
+      @RequestParam(value = "funding_method", required = false) String fundingMethod,
```

Comment:
In this PR's description and the protocol, `funding_method` is a required parameter. Is it supposed to be required?

Reply:
It 's not marked as required because we need to support both `type` and `funding_method` and they are referencing the same thing